docs/tutorials/bitcoin-price-feed/migrations.md

```diff
@@ -101,8 +103,11 @@ can deploy them into a public network using the `--network` flag:
 - `truffle migrate --network=ethereum.goerli` deploys on the Ethereum Görli
   testnet.
   
-You can find a complete list of supported networks in the `addresses` object
-in the `migrations/2_user_migrations.js` file, and also [here][addresses].
+You can find a complete list of supported networks in [here][addresses].
+
+!!! tip "Tip: deploying to EVM-compatible chains others than Ethereum"
+    Just set the `WITNET_EVM_REALM` environment variable to any of 
```

Comment:
I'm thinking that we can actually read the realm from the `--network <realm>.<chain>` argument to make things even simpler :thinking: 

For this suggestion to work the same in all networks, we just need to make sure that users also provide that flag when compiling, e.g.: `yarn compile --network boba.rinkeby`. If they don't provide the flag, truffle will obviously be using the default compiler with the Ethereum settings.

I made this change locally and I can confirm it works. It's basically the same but using a flag instead of an environment variable (which depending on the OS and the tech-savviness of the user, it may pose UX challenges)

Of course using an environment variable has the convenience for, imagine, Boba devs, to always have their projects default to Boba realm.

With all of this in mind, maybe the best thing we can do is `from_env || from_flag || default`. What do you think?